### ISSUE TITLE
Can't always look through discard piles

### ISSUE THREAD

**Issue by YogoGohei** [gryffon/ringteki#761]
I have a Shosuro Actress in play.  My opponent swings with several guys.  He hits a "Rally the Cause" and the "would you like to switch the type" prompt comes up.

I would REALLY like to be able to look through his discard pile at this point to see if I want to switch the conflict type.

**Comment by gryffon** [comment]
Just to double-check, clicking on the discard pile didn't bring up the display with it?

**Comment by YogoGohei** [comment]
correct.  At least not at certain times.  I am not sure of a full list of those times.  Usually it does, but there are times where the window doesn't open.

**Comment by jeremylarner** [comment]
Yeah I had this bug earlier.  It was really weird.  Within a single reaction window, it worked, then it didn't work, then it did

**Comment by YogoGohei** [comment]
I ran into this again and got a screen shot of exactly where we were.  I can Open his Dynasty but not his Conflict.  Clicking his conflict does nothing.  Double clicking it highlights the line of text in the prompt... for some reason.

hope this helps:

![image](https://user-images.githubusercontent.com/32277970/33027282-ae6b49dc-cde0-11e7-9c38-67bb176ad6d2.png)
